packages/editor/src/types/config.ts

```diff
@@ -13,7 +13,10 @@ export type FieldOption<TValue = PrimitiveValue> = {
   value: TValue;
 };
 
+type Subsection = 'General' | 'Styling' | 'Behaviour';
+
 export type BaseField<ComponentProps extends DefaultComponentProps = DefaultComponentProps> = {
+  subsection: Subsection;
```

Comment:
I could of course make the subsection optional in the same way as the section, and if it is undefined, it is automatically assigned to the "General" subsection.